multicluster/cmd/multicluster-controller/memberclusterannounce_webhook_test.go

```diff
@@ -0,0 +1,219 @@
+package main
```

Comment:
missing copyright

Reply:
Done. Move this to a different PR, will address it there.